easybuild/tools/testing.py

```diff
@@ -248,18 +249,21 @@ def upload_test_report_as_gist(test_report, descr=None, fn=None):
     return gist_url
 
 
-def post_easyconfigs_pr_test_report(pr_nr, test_report, msg, init_session_state, success):
-    """Post test report in a gist, and submit comment in easyconfigs PR."""
+def post_pr_test_report(pr_nr, repo, test_report, msg, init_session_state, success):
+    """Post test report in a gist, and submit comment in easyconfigs or easyblocks PR."""
 
     github_user = build_option('github_user')
     pr_target_account = build_option('pr_target_account')
-    pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYCONFIGS_REPO
+    if repo == 'easyconfigs':
+        pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYCONFIGS_REPO
+    else:
+        pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYBLOCKS_REPO
```

Comment:
If we manage to enhance this to framework in the future (test reports that is) then we should protect against that by making this a case-stmt instead (of elif) and error out if repo isn't easyconfigs/easyblocks. That way we'll easily catch this one.